fluss-server/src/test/java/com/alibaba/fluss/server/kv/rocksdb/RocksDBResourceContainerTest.java

```diff
@@ -187,8 +193,237 @@ void testConfigurationOptionsFromConfig() throws Exception {
+        try (RocksDBResourceContainer container =
+                new RocksDBResourceContainer(config, instancePath)) {
+            Statistics statistics = container.getStatistics();
+            assertThat(statistics).isNotNull();
+            assertThat(statistics.isOwningHandle()).isTrue();
+        }
+    }
+
+    @Test
+    void testStatisticsConfigurationConsistency() throws Exception {
+        Configuration config = new Configuration();
+
+        // Test with statistics disabled
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), false);
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            assertThat(container.getStatistics()).isNull();
+        }
+
+        // Test with statistics enabled
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            assertThat(container.getStatistics()).isNotNull();
+        }
+    }
+
+    @Test
+    void testStatisticsMemoryManagement() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+
+        // Create and close many containers to test memory management
+        List<RocksDBResourceContainer> containers = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            RocksDBResourceContainer container = new RocksDBResourceContainer(config, null);
+            containers.add(container);
+
+            // Verify statistics is created and valid
+            Statistics stats = container.getStatistics();
+            assertThat(stats).isNotNull();
+            assertThat(stats.isOwningHandle()).isTrue();
+        }
+
+        // Close all containers
+        for (RocksDBResourceContainer container : containers) {
+            container.close();
+        }
+
+        // This should not cause memory leaks or crashes
+        System.gc(); // Suggest garbage collection
+    }
+
+    @Test
+    void testStatisticsThreadSafety() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            Statistics statistics = container.getStatistics();
+            assertThat(statistics).isNotNull();
+
+            // Test concurrent access to statistics
+            int threadCount = 10;
+            CountDownLatch latch = new CountDownLatch(threadCount);
+            List<Exception> exceptions = Collections.synchronizedList(new ArrayList<>());
+
+            for (int i = 0; i < threadCount; i++) {
+                new Thread(
+                                () -> {
+                                    try {
+                                        // Access statistics concurrently
```

Comment:
I don't see any tests for the concurrency accessing. I think we don't need to add concurrency test just for `RocksDBResourceContainer#getStatistics`.